internal/integration/k8s/version.go

```diff
@@ -0,0 +1,49 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+// +build integration_k8s
+
+package k8s
+
+import (
+	"fmt"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/talos-systems/talos/internal/integration/base"
+	"github.com/talos-systems/talos/pkg/constants"
+)
+
+// VersionSuite verifies Talos version
+type VersionSuite struct {
+	base.K8sSuite
+}
+
+// SuiteName ...
+func (suite *VersionSuite) SuiteName() string {
+	return "k8s.VersionSuite"
+}
+
+// TestExpectedVersion verifies that node versions matches expected
+func (suite *VersionSuite) TestExpectedVersion() {
+	nodes, err := suite.Clientset.CoreV1().Nodes().List(metav1.ListOptions{})
+	suite.Require().NoError(err)
+
+	expectedTalosVersion := fmt.Sprintf("Talos (%s)", suite.Version)
+	expectedContainerRuntimeVersion := fmt.Sprintf("containerd://%s", "1.3.0") // TODO: expose as constant
```

Comment:
how about exposing those versions in `constants`?

it would require bumping each time we get new software version, but at the same time I feel it's critical to verify at least some of these versions to make sure we get what is expected

Reply:
SGTM

Reply:
I'll get it updated for the next version of this PR